commit ff782688919b51476a3021400e229fef075a0237
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Feb 8 18:35:54 2015 +0100

    hurd: fix unwind-resume.c build

commit 28b067571e98d3047da7020014502ff2391703be
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Feb 8 18:46:00 2015 +0100

    hurd: fix unwind-resume.c build

diff --git a/sysdeps/gnu/unwind-resume.c b/sysdeps/gnu/unwind-resume.c
index 267949e..d446804 100644
--- a/sysdeps/gnu/unwind-resume.c
+++ b/sysdeps/gnu/unwind-resume.c
@@ -43,9 +43,13 @@ __libgcc_s_init (void)
     __libc_fatal (LIBGCC_S_SO
                   " must be installed for pthread_cancel to work\n");
 
+#ifdef PTR_MANGLE
   PTR_MANGLE (resume);
+#endif
   __libgcc_s_resume = resume;
+#ifdef PTR_MANGLE
   PTR_MANGLE (personality);
+#endif
   libgcc_s_personality = personality;
 }
 
@@ -57,7 +57,9 @@ _Unwind_Resume (struct _Unwind_Exception *exc)
     __libgcc_s_init ();
 
   __typeof (__libgcc_s_resume) resume = __libgcc_s_resume;
+#ifdef PTR_DEMANGLE
   PTR_DEMANGLE (resume);
+#endif
   (*resume) (exc);
 }
 #endif
@@ -69,6 +71,8 @@ __gcc_personality_v0 PERSONALITY_PROTO
     __libgcc_s_init ();
 
   __typeof (libgcc_s_personality) personality = libgcc_s_personality;
+#ifdef PTR_DEMANGLE
   PTR_DEMANGLE (personality);
+#endif
   return (*personality) PERSONALITY_ARGS;
 }
